csv_detective/detection.py

```diff
@@ -2,9 +2,14 @@
 import numpy as np
 from cchardet import UniversalDetector
 from ast import literal_eval
+import logging
+from time import time
+from csv_detective.utils import print_colored_logs
```

Comment:
rename function and its call to something like `display_logs_depending_process_time`